pmd-apex/src/main/resources/rulesets/apex/style.xml

```diff
@@ -132,4 +132,24 @@ Many interfaces (e.g. Batch) required global modifiers in the past but don't req
     </example>
   </rule>
 
+<rule name="AvoidNonExistentAnnotations"
```

Comment:
rulesets are deprecated. New rules should be added to categories. In particular, this looks like a fit for the [`errorprone` category](https://github.com/pmd/pmd/blob/master/pmd-apex/src/main/resources/category/apex/errorprone.xml)